### PR TITLE
Adds tests to makefile and resolves most make lint errors

### PR DESCRIPTION
**Pull request by Dominik1999**
3 errors remain, but will be solved this week when files are changed

### CODE REVIEW

File: src/raiden_libs/transport/transport.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Transport(gevent.Greenlet):
     """
     def __init__(self):
         super().__init__()
-        self.message_callbacks = list()
+        self.message_callbacks: list = list()
```
Comment (palango) [inline src/raiden_libs/transport/transport.py]: While `list` works, we always try to use classes from the `typing` module as annotations. So `typing.List` here.

File: tests/pathfinding/test_graphs.py [hunk 2]
```diff
@@ -8,16 +8,17 @@
...
-from raiden_libs.types import Address
+from raiden_libs.types import Address, ChannelIdentifier
 
 
 # This test is boring right now, but should get more interesting as the routing
 # gets more options.
 def test_edge_weight(addresses):
-    a = addresses[0]
-    b = addresses[1]
+    a = ChannelIdentifier(1)
```
Comment (palango) [inline tests/pathfinding/test_graphs.py]: can you sue better names here

File: src/raiden_libs/transport/transport.py (hunk 1 above)
Comment (Dominik1999) [reply src/raiden_libs/transport/transport.py]: ` self.message_callbacks: typing.List = list()` ?

Comment (palango) [reply src/raiden_libs/transport/transport.py]: yes, or import list from typing first and then just use it there

File: tests/pathfinding/test_graphs.py (hunk 2 above)
Comment (Dominik1999) [reply tests/pathfinding/test_graphs.py]: done

Comment (palango) [reply tests/pathfinding/test_graphs.py]: Do't see it yet

File: tests/monitoring/conftest.py [hunk 3]
```diff
@@ -4,7 +4,6 @@
 """
 
 from raiden_libs.test.fixtures import *  # noqa
-from raiden_libs.test.fixtures.web3 import *  # noqa
```
Comment (palango) [inline tests/monitoring/conftest.py]: Don't remove this one, but the line before.

**Pull request merged**